controllers/Handle.php

```diff
@@ -90,27 +103,34 @@ public function callback(string $provider): RedirectResponse
             // ]);
             // $user->setSsoConfig('allow_password_auth', false);
             // @TODO: Event here for registering user if desired, default fallback abort behaviour
-            abort(403, 'User not found');
+            return $this->redirectToSigninPage("An account for $email could not be found.");
         }
 
         if (
             $ssoUser->getId()
-            && $user->getSsoId($provider) !== $ssoUser->getId()
+            && $user->getSsoValue($provider, 'id') !== $ssoUser->getId()
         ) {
             // @TODO: Check if request / user is allowed to associate this account to this provider's ID
-            $user->setSsoId($provider, $ssoUser->getId());
+            $user->setSsoValues($provider, ['id' => $ssoUser->getId()]);
             $user->save();
         }
 
         // Check if the user is allowed to keep a persistent session
         if (is_null($remember = Config::get('cms.backendForceRemember', false))) {
-            $remember = false;
-            // @TODO: Get this from the request
-            // $remember = (bool) post('remember');
+            // @TODO: needs to be saved to the Session on the signin form using an ajax request
+            $remember = Session::pull('backend.forceRemember', false);
```

Comment:
Is this implemented?

Reply:
No

Reply:
Let's not have it here then. Functionally the difference between this and the comment is nil except that having this here without the other half of the implementation required is a bit of a code smell to me.

Reply:
gone